packages/venia-ui/lib/components/CartPage/ProductListing/product.css

```diff
@@ -1,11 +1,25 @@
 .root {
+    display: grid;
+    grid-auto-flow: row;
+}
+
+.item {
     align-items: start;
     display: grid;
     grid-gap: 0.5rem 1rem;
     grid-template-areas: 'image details kebab';
     grid-template-columns: 100px 1fr min-content;
 }
 
+.errorText {
+    color: rgb(var(--venia-global-color-error));
+    padding-bottom: 0.5rem;
```

Comment:
Since you have a fancy new grid you're working in, this can be defined by the `.root row-gap` property now. You did go from `0.5rem row-gap + 0.5rem padding-bottom = 1rem total` to just `0.5 padding-bottom`, not sure if that was intentional either.

Reply:
Personally I think `0.5rem` helps attribute the error to the product.